### DISCUSSION TITLE
Android TTS Engines?

### DISCUSSION THREAD

**Discussion by S7venLights** [neonbjb/tortoise-tts#158] [Q&A]
Does anyone know if there are any Android TTS Engines based on Tortoise-tts models?
And any FOSS options for Speech to text?

**Comment by neonbjb** [comment]
I don't think Tortoise will work very well on Android, with the possible exception of those device that have dedicated ML hardware (possibly). It's just too slow.

There are a few open source speech to text engines (for the faster ones capable of on-device inference I'd check out what mycroft uses). If you're looking for a powerful & accurate ASR program, I'd recommend taking a look at my own ocotillo or just wav2vec implementations in general.
